ui/app/models/pki/tidy.js

```diff
@@ -87,6 +87,9 @@ export default class PkiTidyModel extends Model {
   @attr('boolean', { label: 'Tidy the certificate store' })
   tidyCertStore;
 
+  @attr('boolean')
+  tidyCertMetadata;
```

Comment:
I had to add this explicitly to the model instead of relying purely on OpenAPI because the component test was failing due to not calling the path-help method before the test. Alternatively, I could update that test to hydrate the model first, which will make the test less fragile from future attribute additions